packages/icons/src/service-azure-artifacts.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 49 49"><path d="M18.465 48.656H2.402a2.103 2.103 0 01-2.093-2.104V26.75c0-1.16.942-2.094 2.093-2.094h16.063v24z" fill="#B41055"/><path d="M46.255 48.656H39.45v-24h6.806c1.131 0 2.054.923 2.054 2.055V46.6a2.058 2.058 0 01-2.054 2.055z" fill="#E86299"/><path d="M18.613 48.656h21.768v-24H16.39v21.777c0 1.22.992 2.223 2.222 2.223z" fill="#F173A7"/><path d="M38.199.656H10.657c-1.19 0-2.153.963-2.153 2.154v21.846h31.848V2.81A2.151 2.151 0 0038.199.656z" fill="#C12664"/><path d="M32.365.656H10.657c-1.19 0-2.153.963-2.153 2.154v21.846h23.861v-24z" fill="#CF3071"/><path d="M22.522 8.597h-4.167a1.99 1.99 0 01-1.984-1.985 1.99 1.99 0 011.984-1.985h4.167a1.984 1.984 0 110 3.97z" fill="#B5205C"/><path d="M32.313 38.64H20.438v2.035h11.876v-2.034zm-5.119 4.042h-6.756v2.034h6.756v-2.034z" fill="#FE90BE"/><path d="M30.382 32.715h-3.959a2.063 2.063 0 01-2.064-2.065c0-1.141.923-2.064 2.064-2.064h3.959c1.14 0 2.063.923 2.063 2.065a2.063 2.063 0 01-2.063 2.064z" fill="#CB2E6F"/><path d="M14.276 32.715h-3.958a2.063 2.063 0 01-2.064-2.065c0-1.141.923-2.064 2.064-2.064h3.958c1.141 0 2.064.923 2.064 2.065a2.063 2.063 0 01-2.064 2.064z" fill="#900740"/></svg>
```

Comment:
fill="none" should not be present on the SVG element

Reply:
roger